CMakeLists.txt

```diff
@@ -189,8 +189,8 @@ if (MSVC)
 	# Travis CI      : CI=True & TRAVIS=True
 	# Appveyor       : CI=True & APPVEYOR=True
 	# Azure Pipelines: TF_BUILD=True
-	if(DEFINED ENV{TF_BUILD} OR (DEFINED ENV{CI} AND DEFINED ENV{APPVEYOR}))
-		message(STATUS "Running in an Azure or AppVeyor environment, setting -Werror")
+	if(${COMPILE_WARNING_AS_ERROR})
```

Comment:
Probably should be `if(COMPILE_WARNING_AS_ERROR)`

Reply:
fixed